src/common/libflux/attr.c

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
Commit message for 79de48c309af39342080b63437f4affbe8e59bf1
s/fucntion/function/